mtags/src/main/scala-3/scala/meta/internal/pc/CompilerSearchVisitor.scala

```diff
@@ -23,6 +23,9 @@ class CompilerSearchVisitor(
   private def isAccessible(sym: Symbol): Boolean = try
     sym != NoSymbol && sym.isPublic && sym.isStatic
   catch
+    case err: AssertionError =>
+      logger.log(Level.WARNING, err.getMessage())
```

Comment:
Changed to warning since there is nothing we can currently do about it and the user might be scared when seeing SEVERE

Reply:
We create report for `NonFatal` maybe here we'd like one too?

Reply:
Currently this will be the same report a lot of the times. I think it's fine to report a warning in this case so if the exception changes we can see that.

Reply:
Ok. We do have a mechanism of `id`s that can be used to prevent the same error from being reported over and over again but if you expect all of this to be the same error then it doesn't make much sense.